invenio_rdm_records/config.py

```diff
@@ -176,3 +176,10 @@ def _(x):
 RDM_RECORDS_DOI_DATACITE_PASSWORD = ""
 RDM_RECORDS_DOI_DATACITE_PREFIX = "10.1234"
 RDM_RECORDS_DOI_DATACITE_TEST_MODE = True
+
+
+# Invenio-IIIF
+# ===================
+
+IIIF_API_DECORATOR_HANDLER = None
```

Comment:
These two variables shoudl be set in Invenio-App-RDM instead. ``ext.py:init_config()``  is not picking them up, and because it's an extension default, then we don't know if  Invenio-IIIF or Invenio-RDM-Records will set these values first.

Reply:
They are not form this PR... the come from [here](https://github.com/inveniosoftware/invenio-rdm-records/pull/599)... and yes... when testing we were adding them to the invenio.cfg file... but I put them there to make sure is understood what the handler is for... can remove, but is part of another PR not this one :)